src/Buzzer.cpp

```diff
@@ -12,7 +12,6 @@ Buzzer::Buzzer(){
 }
 
 void Buzzer::soundTone(int frequency) {
-  if(Properties::DEBUGGING_ACTIVE) Serial.println("Buzzer has started sounding");
```

Comment:
Not it wasn't intentional. I'm going to scrap the request for the pull entirely as it's not cleary up to par, and use your comments as notes, on redoing it. 

I'm also going to add a problem for documenting the code thoroughly so I'm able to understand it better and push it so if others look at it they can understand as well.